dam/loans/models.py

```diff
@@ -28,8 +30,13 @@ class ItemReservation(models.Model):
 
 
 class ItemLoan(models.Model):
+    def get_duration():
```

Comment:
```suggestion
    def get_duration(self):
```
Sorry for the earlier suggestions, previous versions should be good

Reply:
No worries!